fbw-a380x/src/systems/instruments/src/PFD/PFD.tsx

```diff
@@ -57,6 +61,15 @@ export class PFDComponent extends DisplayComponent<PFDProps> {
 
   private failuresConsumer;
 
+  private readonly fwcFlightPhase = ConsumerSubject.create(this.sub.on('fwcFlightPhase').whenChanged(), 0);
+
+  private readonly groundSpeed = ConsumerSubject.create(this.sub.on('groundSpeed').whenChanged(), 0);
+
+  private readonly raHeight = ConsumerSubject.create(this.sub.on('chosenRa').whenChanged(), Arinc429Word.empty());
+
+  // FIXME add touch&go condition
+  private pitchTrimIndicatorVisible = MappedSubject.create(([flightPhase, gs, raHeight]) => (flightPhase >= 9 && gs < 30) || (flightPhase < 6 && raHeight.valueOr(0) < 50), this.fwcFlightPhase, this.groundSpeed, this.raHeight);
```

Comment:
If I had to guess, I'd say that this probably doesn't use the FWC flight phase IRL. More likely I think it's a simple flip-flop with the RA height and the GS. Of course this is still just guessing, but experience shows that the flight phase is rarely used outside of the FWS itself

Reply:
But the flip-flop would then reset, if I were to re-load the instrument in flight, right? 
I think for maintainability using the FWC flight phase would have some advantages.

We would anyway have to improve the display conditions in another PR, this doesn't take touch&gos into account right now.

Reply:
Not necessarily, for example, take the following conditions:
```
if (ra > 50) {
    showTrim = false;
} else if (gs < 30) {
    showTrim = true;
}
```
and showTrim would be initialized to false. If I'm not totally overlooking something I think this should behave as expected. But in any case, after looking at some more references I'm doubting that any of the two solutions are correct

Reply:
No it would not, the variable would need to be a field of the class of course to be persistent. Otherwise it would not be a flip-flop. So if you stay below 50ft and go above 30kts, it will retain the last value which would be `true`. Maybe my example didn't make that clear enough

Reply:
> No it would not, the variable would need to be a field of the class of course to be persistent. Otherwise it would not be a flip-flop. So if you stay below 50ft and go above 30kts, it will retain the last value which would be `true`. Maybe my example didn't make that clear enough

Ah, you're right, didn't think about the persistent state. Sounds like a good solution, which is easily extensible for touch&go cases. There's one corner case (take-off roll above 30kts, but still on ground), where an instrument reload would lead to the indicator not showing, but that's negligible.

Reply:
I updated the conditions, and also added a condition for touch&gos